### DISCUSSION TITLE
Parte 1 enter ID N

### DISCUSSION THREAD

**Discussion by mt-garcia** [IIC2133-PUC/2024-2#21] [Tarea 0] [answered]
Hola!

En caso de que la tienda esté llena, es necesario imprimir 
Penguin {ID_penguin} has entered with {N} puffles ?
Además de que la tienda está en su máxima capacidad?
Porque no tendría sentido de que entre a la tienda si es que esta está llena.
Gracias de antemano!

**Comment by sebacarrascop** [comment, marked as answer]
hola! si está llena no debe imprimir eso
